pipelines/py/collect-wheels.yaml

```diff
@@ -0,0 +1,28 @@
+name: Collect wheels built by a python pipeline
+
+needs:
+  packages:
+    - util-linux
+
+inputs:
+  dest:
+    description: the destination
+    default: ${{targets.subpkgdir}}/usr/share/wheels
+
+pipeline:
```

Comment:
📖 what's the purpose of defining a pipeline here instead of in melange? specific use cases just for wolfi-dev/os?

Reply:
ahh is this what you've been calling bootstrapping? helps with any circular dependencies?

Reply:
> what's the purpose of defining a pipeline here instead of in melange? specific use cases just for wolfi-dev/os?

pipelines here (call them "local pipelines")
a. are more easily adaptable (just change them in a PR versus 1. change melange, 2. land melange, 3. update systems .... 
b. not globally available (we can't use them in other places without copy/paste)

I feel like "local pipelines" should be used at least until the point at which they've proven very stable.

Reply:
> ahh is this what you've been calling bootstrapping? helps with any circular dependencies?

The `pip-build-install-bootstrap` pipeline that is here locally is something i don't think i would submit to melange. we only need it to be able to build packages that are inter-dependent on each other (setuptools, pip, toml, .... ).  Those are painful to build incrementally.  

This pipeline and the pip-zipapp.yaml are what i meant by bootstrapping. 

Adding the `pip-zipapp.yaml` is really